### PR TITLE
Prune dependencides

### PR DESCRIPTION
**Pull request by emilk**
### What
See commit log

Found with `cargo machete`

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested [demo.rerun.io](https://demo.rerun.io/pr/3490) (if applicable)

- [PR Build Summary](https://build.rerun.io/pr/3490)
- [Docs preview](https://rerun.io/preview/8ca164376fc491e13a67cceb3de5f845676289bb/docs) 
- [Examples preview](https://rerun.io/preview/8ca164376fc491e13a67cceb3de5f845676289bb/examples) 
- [Recent benchmark results](https://ref.rerun.io/dev/bench/)
- [Wasm size tracking](https://ref.rerun.io/dev/sizes/)

### CODE REVIEW

File: crates/re_space_view_bar_chart/Cargo.toml [hunk 1]
```diff
@@ -29,4 +29,3 @@ re_viewer_context.workspace = true
 
 egui_plot.workspace = true
 egui.workspace = true
-vec1.workspace = true
```
Comment (Wumpf) [inline crates/re_space_view_bar_chart/Cargo.toml:32]: any use of vec1 left? Might be able to remove it from the workspace Cargo.toml now (?)

Comment (emilk) [reply crates/re_space_view_bar_chart/Cargo.toml:32]: `crates/re_time_panel/src/time_axis.rs` still uses it

**Review by Wumpf — APPROVED** [review]
neat

**Comment by emilk** [comment]
The build-error is from the red `main`

**Pull request merged**